apps/hash-api/src/graph/ontology/primitive/property-type.ts

```diff
@@ -91,7 +90,7 @@ export const getPropertyTypeById: ImpureGraphFunction<
         },
       },
     })
-    .then(({ data }) => mapSubgraph(data) as Subgraph<PropertyTypeRootType>);
+    .then(({ data }) => data as Subgraph<PropertyTypeRootType>);
```

Comment:
```suggestion
    .then(({ data: subgraph }) =>
      getRoots(subgraph as Subgraph<PropertyTypeRootType>),
    );
```

Reply:
Applied in [Apply suggestions from Code Review](https://github.com/hashintel/hash/pull/2110/commits/460911c284da89861487b718d9a50812321df97f)